src/Lykke.Service.BlockchainCashoutPreconditionsCheck.Core/Domain/Validation/CashoutModel.cs

```diff
@@ -7,7 +7,11 @@ namespace Lykke.Service.BlockchainCashoutPreconditionsCheck.Core.Domain.Validati
     public class CashoutModel
     {
         public string AssetId { get; set; }
+
         public decimal Volume { get; set; }
+
         public string DestinationAddress { get; set; }
+
+        public string DestinationAddressBase { get; set; }
```

Comment:
This field really should be added to the contract? this service could use BlockchainWallets API to check if given blockchain support address extensions and extract base and extension address parts. This should be already done in the BlockchainCashoutPreconditionsCheck in the LWDEV-6879

Reply:
fixed